src/module.ts

```diff
@@ -193,48 +194,48 @@ export enum UnaryOp {
   AllTrueI8x16 = 70 /* _BinaryenAllTrueVecI8x16 */,
   BitmaskI8x16 = 71 /* __BinaryenBitmaskVecI8x16 */,
   AbsI16x8 = 72 /* __BinaryenAbsVecI16x8 */,
-  NegI16x8 = 73 /* _BinaryenNegVecI16x8 */,
-  AnyTrueI16x8 = 74 /* _BinaryenAnyTrueVecI16x8 */,
-  AllTrueI16x8 = 75 /* _BinaryenAllTrueVecI16x8 */,
-  BitmaskI16x8 = 76 /* _BinaryenBitmaskVecI16x8 */,
-  AbsI32x4 = 77 /* _BinaryenAbsVecI32x4 */,
-  NegI32x4 = 78 /* _BinaryenNegVecI32x4 */,
-  AnyTrueI32x4 = 79 /* _BinaryenAnyTrueVecI32x4 */,
-  AllTrueI32x4 = 80 /* _BinaryenAllTrueVecI32x4 */,
-  BitmaskI32x4 = 81 /* _BinaryenBitmaskVecI32x4 */,
-  NegI64x2 = 82 /* _BinaryenNegVecI64x2 */,
-  AnyTrueI64x2 = 83 /* _BinaryenAnyTrueVecI64x2 */,
-  AllTrueI64x2 = 84 /* _BinaryenAllTrueVecI64x2 */,
-  AbsF32x4 = 85 /* _BinaryenAbsVecF32x4 */,
-  NegF32x4 = 86 /* _BinaryenNegVecF32x4 */,
-  SqrtF32x4 = 87 /* _BinaryenSqrtVecF32x4 */,
-  CeilF32x4 = 88 /* _BinaryenCeilVecF32x4 */,
-  FloorF32x4 = 89 /* _BinaryenFloorVecF32x4 */,
+  NegI16x8 = 74 /* _BinaryenNegVecI16x8 */,
```

Comment:
Another one missing here

Reply:
This one is strange. Order should be: https://github.com/WebAssembly/binaryen/blob/68294338a1cc7337e808671e75933b1134d18a90/src/wasm.h#L163-L166

```
BitmaskVecI8x16
PopcntVecI8x16 // new
AbsVecI16x8
NegVecI16x8
```

meaning something is off here since the missing one is before AbsVecI16x8, not between AbsVecI16x8 and NegVecI16x8 hmm.